Source/VolunteerReporting/Read/AutomaticReplyMessages/IAutomaticReplyKeyMessages.cs

```diff
@@ -0,0 +1,16 @@
+using Concepts;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Read.AutomaticReplyMessages
+{
+    public interface IAutomaticReplyKeyMessages
+    {
+        Task<AutomaticReplyKeyMessage> GetByProjectTypeLanguageAndHealthRiskAsync(Guid projectId, AutomaticReplyKeyMessageType type, string language, Guid healthRiskId);
```

Comment:
I'm slightly confused by this method name, is it considered best practice to name read model methods by listing parameters?

Reply:
Not at all :) I'd say it could be simply named Get. I'll try to clean up on first occasion